documentation/modules/exploit/multi/fileformat/libreoffice_logo_exec.md

```diff
@@ -0,0 +1,90 @@
+## Description
```

Comment:
```suggestion
## Introduction
```

Reply:
The documentation template does not indicate whether `Introduction` or `Description` is preferred. `Description` Is preferred for consistency.

```
root@kali-2018:~/Desktop/metasploit-framework# grep -rn "# Introduction" documentation/ | wc -l
22
root@kali-2018:~/Desktop/metasploit-framework# grep -rn "# Description" documentation/ | wc -l
255
```